config/dev.exs

```diff
@@ -51,7 +51,7 @@ config :archethic, ArchEthic.P2P.BootstrappingSeeds,
   genesis_seeds:
     System.get_env(
       "ARCHETHIC_P2P_BOOTSTRAPPING_SEEDS",
-      "127.0.0.1:3002:00011D967D71B2E135C84206DDD108B5925A2CD99C8EBC5AB5D8FD2EC9400CE3C98A:tcp"
+      "127.0.0.1:3002:4000:00011D967D71B2E135C84206DDD108B5925A2CD99C8EBC5AB5D8FD2EC9400CE3C98A:tcp"
```

Comment:
I don't think we need the HTTP port there, as this information is just for bootstrapping seed where only the P2P port is required

Reply:
If we don't we will get some serialization and deserialization errors.
We used `String.split` for splitting based on the `:` so, we expect `http_port` over there

Reply:
We may assume the default HTTP port from the configuration for the bootstrapping nodes.
Then when the node will join the network the information will be updated, when it will get other nodes information.
Don't you think ?

Reply:
but, when we split with `:` we miss the order right?

Reply:
Why don't you let: `127.0.0.1:3002:00011D967D71B2E135C84206DDD108B5925A2CD99C8EBC5AB5D8FD2EC9400CE3C98A:tcp` and  in the Node struct, let the HTTP be the default one from the configuration ?

Reply:
Yeah Updated ...